ADMVEDU/Network/NetworkError.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  NetworkError.swift
+//  ADMVEDU
+//
+//  Created by Bobr, Andrey on 4.05.21.
+//
+
+import Foundation
+
+enum NetworkError: Error {
+    case notFound
+    case inavlidData
+    case unexpected(code: Int)
+
+}
+
+extension NetworkError {
+    var isFatal: Bool {
+        if case NetworkError.unexpected = self { return true }
+        else { return false }
```

Comment:
1. if ... {
} else {
}

2. would be better to return a single condition instead of if/else

Reply:
> * if ... {
>   } else {
>   }
> * would be better to return a single condition instead of if/else

How to replace this with an equivalent single condition?